memoized_lcsh.txt

```diff
@@ -1 +1 @@
-{"Semantic Web"=>"https://id.loc.gov/authorities/subjects/sh2002000569", "cancer"=>"https://id.loc.gov/authorities/subjects/sh85000095"}
\ No newline at end of file
+{"Semantic Web"=>"https://id.loc.gov/authorities/subjects/sh2002000569", "cancer"=>"https://id.loc.gov/authorities/subjects/sh85000095", "Cancer"=>"http://id.loc.gov/authorities/subjects/sh85019492"}
```

Comment:
These caches had to be destroyed to make sure we were getting correct values.

Reply:
Good point. I did a test run of the exporter to make sure things were not breaking yesterday. I'll plan on doing that again after finishing all my changes then I can copy the cache file to my local instance and include it in the commit.